srv/salt/_modules/wait.py

```diff
@@ -74,30 +79,32 @@ def _check_status(self, current):
         """
         Return the "correct" matching status
         """
+        # pylint: disable=no-else-return
         if self.settings['negate']:
             log.debug("status != {}".format(self.settings['status']))
-            return (current != self.settings['status'])
+            return current != self.settings['status']
         else:
             log.debug("status == {}".format(self.settings['status']))
-            return (current == self.settings['status'])
+            return current == self.settings['status']
 
 
+# pylint: disable=too-few-public-methods
 class FsStatusCheck(HealthCheck):
     """
     Check the fsmap status of the ceph status output. Wait till all active MDS's
     daemons have reached up:active status
     """
 
-    def __init__(self, **kwargs):
-        super(FsStatusCheck, self).__init__(**kwargs)
-
```

Comment:
I believe this super had a purpose. E.g. not to inherit stuff from parent class?
Or similar: https://stackoverflow.com/questions/576169/understanding-python-super-with-init-methods

I decided not to touch it in my PR.

@swiftgist can you please elaborate on this super?

Reply:
The code is mine. The purpose of the call to super was to have the constructor of the base class called. There is no reason to explicitly call the super constructor, its just inherited....right?

Reply:
Right! Just super(FsStatusCheck, self).__init__(**kwargs) looked strange to me, so i assumed it does some mega magic.

Reply:
The only magic was that this code was supposed to be python 2 and 3 compatible.

Reply:
@jan--f right, that's py2/3 compatibility. That can stay

Reply:
@jschmid1 shouldn't it be fine like it is now? since I don't actually do any work in the child constructor, it can simply be left out, resulting in the base class's constructor being called. Or amI missing something?

Reply:
here's the proof: http://paste.opensuse.org/35677684

Reply:
uhm thats not equivalent code. I removed the children's constructors completely. I.e. the base class' constructor will be used without me redirecting things. http://paste.opensuse.org/95356189

Reply:
yikes, my bad. All good then.